### PR TITLE
advertise the new translation platform

### PR DESCRIPTION
**Pull request by Jibec**
you may want to simplify the pull-translations file as you don't have to manually pull from zanata anymore

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -46,3 +46,8 @@ and install ABRT can be found in [INSTALL.md](INSTALL.md)
 * [D-Bus](https://www.freedesktop.org/wiki/Software/dbus/)
 * [SELinux](https://github.com/SELinuxProject/selinux/wiki)
 * [systemd](https://www.freedesktop.org/wiki/Software/systemd/)
+
+
+### Translations
+
+[![Translation status](https://translate.fedoraproject.org/widgets/abrt/-/287x66-grey.png)](https://translate.fedoraproject.org/engage/abrt/?utm_source=widget)
```
Comment (ernestask) [inline README.md]: This is wrong.

```
[![Translation status](https://translate.fedoraproject.org/widgets/abrt/-/abrt/svg-badge.svg)](https://translate.fedoraproject.org/engage/abrt/?utm_source=widget)
```

And maybe even without the tracking:

```
[![Translation status](https://translate.fedoraproject.org/widgets/abrt/-/abrt/svg-badge.svg)](https://translate.fedoraproject.org/engage/abrt/)
```

**Comment by ernestask** [comment]
I would separate these two changes. We neither need to mention Zanata nor use the script for pulling ever again.

File: README.md (hunk 1 above)
Comment (ernestask) [inline README.md]:
```suggestion
[![Translation status](https://translate.fedoraproject.org/widgets/abrt/-/abrt/svg-badge.svg)](https://translate.fedoraproject.org/engage/abrt/)
```

Comment (ernestask) [reply README.md]: Also maybe put it on top? You now have to scroll *all* the way down to see the badge.

Comment (ernestask) [reply README.md]: These heading would also have to go away.

**Comment by mkutlak** [comment]
@trapas 
add to whitelist

**Comment by ernestask** [comment]
Amended and pushed directly.

**Pull request closed without merging**